framework/sac_rlhf.py

```diff
@@ -253,7 +255,40 @@ def load_replay_buffer(replay_buffer: ReplayBuffer, path: str):
     replay_buffer.ground_truth_rewards = buffer_data["ground_truth_rewards"]
     replay_buffer.dones = buffer_data["dones"]
 
-    logging.debug(f"Replay buffer loaded from {path}")
+    logging.info(f"Replay buffer loaded from {path}")
+
+
+def uniform_sampling(rb: ReplayBuffer, reward_net: RewardNet):
+    traj_mb1, traj_mb2 = rb.sample_trajectories()
+    return traj_mb1[0], traj_mb2[0]
+
+
+def disagreement_sampling(rb: ReplayBuffer, reward_net: RewardNet):
+    traj_mb1, traj_mb2 = rb.sample_trajectories()
+
+    disagrees = []
```

Comment:
Add `with torch.no_grad():` before for loop to avoid unnecessary calculation of gradients